sdk/mixedreality/mixedreality-authentication/package.json

```diff
@@ -112,6 +112,18 @@
     "typedoc": "0.15.2"
   },
   "//sampleConfiguration": {
-    "skipFolder": true
+    "skipFolder": true,
+    "productName": "Azure Mixed Reality Authentication",
+    "productSlugs": [
+      "azure",
+      "object-anchors",
+      "remote-rendering",
+      "spatial-anchors"
+    ],
+    "requiredResources": {
+      "Azure Object Anchors account": "https://docs.microsoft.com/azure/object-anchors/quickstarts/get-started-model-conversion#create-an-object-anchors-account",
+      "Azure Remote Rendering account": "https://docs.microsoft.com/azure/remote-rendering/quickstarts/convert-model#azure-setup",
+      "Azure Spatial Anchors account": "https://docs.microsoft.com/en-us/azure/spatial-anchors/quickstarts/get-started-hololens?tabs=azure-portal#create-a-spatial-anchors-resource"
```

Comment:
@witemple-msft Same kind of thing here. An account from one of these is required.

Reply:
There's a bit of an undocumented feature that will allow you to insert arbitrary Markdown text after this paragraph. You can see how this feature is used in the description of this PR: https://github.com/Azure/azure-sdk-for-js/pull/14591

You could create a snippet that includes some text like

```markdown
Any one of the above Azure resources can be used with the samples. Set the `MIXEDREALITY_ACCOUNT_DOMAIN`
environment variable to the domain corresponding to the account (of any of the above resource types) you
wish to use when running the samples.
```

And then set the path to that snippet in `["//sampleConfiguration"].customSnippets.prerequisites` in the package.json to tweak the instructions.

Reply:
@witemple-msft Thanks for the suggestion. I've updated it with the snippet. Please take a look and let me know if that looks right to you.